src/ring_theory/dedekind_domain/adic_valuation.lean

```diff
@@ -333,3 +333,43 @@ instance adic_completion.has_lift_t : has_lift_t K (v.adic_completion K) :=
 def adic_completion_integers : valuation_subring (v.adic_completion K) := valued.v.valuation_subring
 
 end is_dedekind_domain.height_one_spectrum
+
+variables (R K)
+
+lemma adic_completion.is_integer (x : v.adic_completion K) :
+  x ∈ v.adic_completion_integers K ↔ (valued.v x : with_zero (multiplicative ℤ)) ≤ 1 := by refl
```

Comment:
If you merge master then you should have access to the notation introduced in #14064
```suggestion
  x ∈ v.adic_completion_integers K ↔ (valued.v x : ℤₘ₀) ≤ 1 := by refl
```

Reply:
I presume the proof can also be `iff.rfl` instead of `by refl`?

Reply:
Right, I saw this got merged a couple days ago! Kevin told me you were working on this, thanks for PRing it.